resources/fetchersManager/factory/k8s_factory.go

```diff
@@ -0,0 +1,67 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package factory
+
+import (
+	"github.com/elastic/cloudbeat/config"
+	"github.com/elastic/cloudbeat/resources/conditions"
+	"github.com/elastic/cloudbeat/resources/fetchers"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"github.com/elastic/cloudbeat/uniqueness"
+	"github.com/elastic/elastic-agent-libs/logp"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+var vanillaFsPatterns = []string{
+	"/hostfs/etc/kubernetes/scheduler.conf",
+	"/hostfs/etc/kubernetes/controller-manager.conf",
+	"/hostfs/etc/kubernetes/admin.conf",
+	"/hostfs/etc/kubernetes/kubelet.conf",
+	"/hostfs/etc/kubernetes/manifests/etcd.yaml",
+	"/hostfs/etc/kubernetes/manifests/kube-apiserver.yaml",
+	"/hostfs/etc/kubernetes/manifests/kube-controller-manager.yaml",
+	"/hostfs/etc/kubernetes/manifests/kube-scheduler.yaml",
+	"/hostfs/etc/systemd/system/kubelet.service.d/10-kubeadm.conf",
+	"/hostfs/etc/kubernetes/pki/*",
```

Comment:
Double star is important here

```suggestion
	"/hostfs/etc/kubernetes/pki/**",
```

See https://github.com/elastic/cloudbeat/pull/981

Reply:
I had forgotten to fix the integration repo, see: https://github.com/elastic/integrations/pull/6622